src/main/java/hu/wurfel/refference/school/diary/DiaryController.java

```diff
@@ -1,40 +1,36 @@
 package hu.wurfel.refference.school.diary;
 
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.ArrayList;
 
-@RestController
+@RestController("/api/v1/Diary")
 public class DiaryController {
 
 
-    @Autowired
     DiaryService diaryService;
 
-    @PostMapping("/api/v1/search/Diary")
+    @PostMapping("/search")
```

Comment:
You mean....
```
@PostMapping("/diaries")
```
? 😄

Reply:
Oh, no, értem miért ezt csináltad. Akkor viszont 
```
@PostMapping("/diaries/students")
```

Reply:
szerintem azóta megváltozott, igy ez már nem releváns